webpack/fills_index.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import SystemStatuses from './components/extensions/about/SystemStatuses';
+import { AddGlobalFill } from 'foremanReact/components/common/Fill/GlobalFill';
+
+AddGlobalFill('about-footer-slot', 'katelo-about-system-statuses', <SystemStatuses />, 100);
```

Comment:
All other states use lowerCamelCase, could we unify it?  
![](https://user-images.githubusercontent.com/2664090/54910638-07684380-4eed-11e9-9a6e-8d1c54174fbf.png)

Reply:
I agree with this comment.